src/views/components/probe-single-card.html

```
@@ -0,0 +1,57 @@
+				<span class="probe-single-card_details_info_city_name">
+					{{location.city}}, {{location.country}}
+				</span>
+			</div>
+			<span class="probe-single-card_details_info_location">
+				{{location.network}}
+			</span>
+		</div>
+	</div>
+	<div class="probe-single-card_tag">
+		{{#each tags: index}}
+			{{#if index < maxTagCount}}
```

HTMLHint (1.5.0)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)